installing/installing_rhv/preparing-to-install-on-rhv.adoc

```diff
@@ -0,0 +1,26 @@
+[id="preparing-to-install-on-rhv"]
+= Preparing to install on RHV
+include::modules/common-attributes.adoc[]
+:context: preparing-to-install-on-rhv
+
+toc::[]
+
+[id="preparing-to-install-on-rhv-prerequisites"]
+== Prerequisites
+
+* You reviewed details about the xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update] processes.
+
+* You read the documentation on xref:../../installing/installing-preparing.adoc#installing-preparing[selecting a cluster installation method and preparing it for users].
+
+[id="choosing-an-method-to-install-ocp-on-rhv"]
+== Choosing a method to install {product-title} on RHV
+
+You can install {product-title} on {rh-virtualization-first} by using one of the following deployment methods.
+
+* **xref:../../installing/installing_rhv/installing-rhv-default.adoc#installing-rhv-default[Installing a cluster quickly on RHV]**: You can install {product-title} on {rh-virtualization} infrastructure that is provisioned by the OpenShift Container Platform installation program. You can install a cluster quickly by using the default configuration options.
```

Comment:
@emarcusRH can you please confirm if this bullet point is correct from an RHV perspective? Thanks!

Reply:
After reviewing each of the OCP on RHV installation assemblies further, I have updated each of the descriptions to be more explicit.